lib/engine/step/tokener.rb

```diff
@@ -33,6 +33,8 @@ def token_cost_override(_entity, _city_hex, _slot, _token)
 
       def available_tokens(entity)
         token_holder = entity.company? ? entity.owner : entity
+        return [] if token_holder.player?
```

Comment:
is this correct?

Reply:
Does not seem to be needed after all, so I have now removed this change.